roxFile/createRoxFile-service.ts

```diff
@@ -0,0 +1,130 @@
+import * as PH from "processhub-sdk";
+import * as Types from "./roxtraFileAPITypes";
```

Comment:
I am not sure if this capital letter import here and in the following line will become an issue as it was in the sdk